applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -217,25 +217,25 @@ endmenu # Stream
 #----------------------------------------------------------------------------#
 menu "Log levels"
 
-config LOG_AUDIO_SYSTEM_LEVEL
-	int "Log level for audio system"
-	default 3
+module = AUDIO_SYSTEM
+module-str = audio-system
+source "subsys/logging/Kconfig.template.log_config"
 
-config LOG_SW_CODEC_SELECT_LEVEL
-	int "Log level for sw_codec_select"
-	default 3
+module = SW_CODEC_SELECT
+module-str = sw-codec-select
+source "subsys/logging/Kconfig.template.log_config"
 
-config LOG_STREAMCTRL_LEVEL
-	int "Log level for streamctrl"
-	default 3
+module = STREAMCTRL
+module-str = streamctrl
+source "subsys/logging/Kconfig.template.log_config"
 
-config LOG_AUDIO_DATAPATH_LEVEL
-	int "Log level for audio_datapath"
-	default 3
+module = AUDIO_DATAPATH
+module-str = audio-datapath
+source "subsys/logging/Kconfig.template.log_config"
 
-config LOG_AUDIO_SYNC_TIMER_LEVEL
-	int "Log level for audio_sync_timer"
-	default 2
+module = AUDIO_SYNC_TIMER
```

Comment:
I cannot see that log level 2 here is propagated into this new method.

Reply:
I have actually fixed issues with levels usage, all app is now using the default level, see the new commits.

Reply:
Yes, and that is nice. Though, earlier, we used various log levels, and I want to make sure that the LOG output remains the same after we merge this PR.

Reply:
If I am not mistaken, we will have INF prints then from this module which were not enabled before.

Reply:
`audio_sync_timer.c` only uses LOG_ERR and LOG_DBG, so using either level 2 or 3 is equivalent.